public/js/services/receiptService.js

```diff
@@ -40,7 +40,36 @@ angular.module('BibBox').service('receiptService', ['$q', 'tmhDynamicLocale', 'p
     }
 
     /**
-     * States receipt.
+     * Get mails address for a list of patrons.
+     *
+     * @param {array} patronIdentifiers
+     *   The identifiers for the patrons.
+     *
+     * @returns {Function}
+     */
+    this.getPatronsInformation = function getPatronsInformation(patronIdentifiers) {
+      var deferred = $q.defer();
+
+      proxyService.once('notification.response', function (mailAddresses) {
```

Comment:
No this is how all the notifications are made. It's outside this PR's scope to re-do that